openlibrary/core/waitinglist.py

```diff
@@ -156,40 +155,6 @@ def update(self, **kw):
         dict.update(self, kw)
 
 
-class Stats:
```

Comment:
I could only find this being used as WLStats in waitinglist_admin, and removed it as unused code